tools/build/install.sh

```diff
@@ -1,8 +1,14 @@
-#!/bin/bash
+#!/bin/sh
 
 set -e
 
 SCRIPT=$(readlink -f "$0")
 BASEDIR=$(dirname "$SCRIPT")
+OS_RELEASE=$(cat /etc/os-release | grep "^ID=" | cut -d= -f2)
 
-bash ${BASEDIR}/node.sh
+if [ $OS_RELEASE = "debian" ]
+then
+    bash ${BASEDIR}/node-${OS_RELEASE}.sh
```

Comment:
Can we just invoke the script? Then it can be written in whatever language one would like to:

```
./${BASEDIR}/node-${OS_RELEASE}
```

Reply:
(also, why is bash required on debian?)

Reply:
because of nvm.sh?

Reply:
Yup, `nvm.sh` has to be `source`'d

Reply:
`source` is available in `sh` as well though. The only reason I see is that nvm.sh is not `sh` compatible and requires `bash`.

Reply:
Yeah, sorry, I meant that. nvm.sh has pragmas that don't work in sh

Reply:
Sooo ./${BASEDIR}/ doesn't work, as the `source` call later is outside of scope. Even with the `#!/bin/bash` I think it's still calling `sh`, which will cause an issue. 

Keeping it forked ensures we call the interpreter we want, and things work properly.

Reply:
This should not be the case, how did you call it? Invoking a script forks it for sure, did you add an additional ‘. ‘ in front?

Reply:
```$ cat tools/build/install.sh | grep node-
./${BASEDIR}/node-${OS_RELEASE}.sh
$ docker build .
...
/stack/boilerplate/install.sh: 9: /stack/boilerplate/install.sh: .//stack/boilerplate/node-debian.sh: not found
```

Reply:
Try removing the first ‘/‘

Reply:
Same result
```
/stack/boilerplate/install.sh: 9: /stack/boilerplate/install.sh: ./stack/boilerplate/node-debian.sh: not found
```